src/Tizen.NUI.Components/Controls/NumberPicker.cs

```diff
@@ -0,0 +1,510 @@
+﻿/* Copyright (c) 2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace Tizen.NUI.Components
+{
+    /// <summary>
+    /// NumberPickerValueChangedEventArgs is a class to notify changed NumberPicker value argument which will sent to user.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class NumberPickerValueChangedEventArgs : EventArgs
+    {
+        /// <summary>
+        /// NumberPickerValueChangedEventArgs default constructor.
+        /// <param name="value">value of NumberPicker.</param>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]   
+        public NumberPickerValueChangedEventArgs(int value)
+        {
+            Value = value;
+        }
+
+        /// <summary>
+        /// NumberPickerValueChangedEventArgs default constructor.
+        /// <returns>The current value of NumberPicker.</returns>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]   
+        public int Value { get; }
+        
+    }
+
+    /// <summary>
+    /// NumberPicker is a class which privides a function that allows the user to select 
+    /// a value through a scrolling mothion by expressing the specified value as a list.
```

Comment:
check spelling, privides / mothion.
you can check spelling easily using 'Visual Studio Spell Checker ' in extension.

Reply:
nice catch thanks